setup.py

```diff
@@ -39,12 +39,13 @@
     packages=find_packages(exclude=('tests',)),
     install_requires=[
         'requests >= 1.2.1, < 3.0',
+        'tqdm',
         FTFY_DEP
     ],
+    tests_require=['pytest', 'requests-mock'],
     entry_points={
         'console_scripts': [
             'lumi-api = luminoso_api.v5_cli:main',
-            'lumi-upload = luminoso_api.v4_upload:main',
```

Comment:
So, one reason why it's probably a good idea to get you to a point where you can run the tests in lumi_stack_tools is that there's a test in lumi_stack_tools that tests this lumi-upload script, and it stops working if you remove this entry point.  I'm not sure what the motivation is for removing this or the line below it.

Reply:
The motivation is not to have confusing scripts around that use the wrong version of the API.

I'll put back lumi-upload for now, but I believe that test should be removed from lumi_stack_tools, as I think I'm the main person pushing for an upload script to exist, and the tests requiring it to use v4 seems wrong.